### ISSUE TITLE
Support Procgen

### ISSUE THREAD

**Issue by KarlXing** [KarlXing/RLCodebase#5]
I plan to add support for procgen training.

- [x] Impala model
- [x] Procgen environment wrapper
- [x] Gradient accumulation for large batch size
- [x] Do training
- [x] Hyperparameter tuning
